### ISSUE TITLE
Make all python code python3 compatible

### ISSUE THREAD

**Issue by amboxer21** [amboxer21/SSHMonitor#12]
All of the python code was written in python2 and needs to be made compatible with python3 - essentially converting all python code.

**Comment by rescenic** [comment]
Please, and thank you.

**Comment by amboxer21** [comment]
This issue has been resolved in branch https://github.com/amboxer21/SSHMonitor2.7/tree/SSHMonitor3

**NOTE:** This branch will **NOT** be merged into master. Instead in the future, the repo will be renamed to SSHMonitor and will work with both Python2 and Pyhton3.

### Checking out SSHMonitor3 branch

#### If you already have a cloned repo on your localhost
```javascript
anthony@ghost:~/Documents/SSHMonitor2.7$ git branch --track SSHMonitor3 origin/SSHMonitor3
Branch 'SSHMonitor3' set up to track remote branch 'SSHMonitor3' from 'origin'.
anthony@ghost:~/Documents/SSHMonitor2.7$ git checkout SSHMonitor3
Switched to branch 'SSHMonitor3'
Your branch is up to date with 'origin/SSHMonitor3'.
anthony@ghost:~/Documents/SSHMonitor2.7$ git branch
* SSHMonitor3
  master
anthony@ghost:~/Documents/SSHMonitor2.7$ 
```

#### Checking out the SSHMonitor3 branch **ONLY** 

```javascript
anthony@ghost:~/Documents$ git clone git@github.com:amboxer21/SSHMonitor2.7.git -bSSHMonitor3
Cloning into 'SSHMonitor2.7'...
remote: Enumerating objects: 15, done.
remote: Counting objects: 100% (15/15), done.
remote: Compressing objects: 100% (15/15), done.
remote: Total 1009 (delta 6), reused 1 (delta 0), pack-reused 994
Receiving objects: 100% (1009/1009), 3.76 MiB | 10.79 MiB/s, done.
Resolving deltas: 100% (586/586), done.
anthony@ghost:~/Documents$ cd SSHMonitor2.7/
anthony@ghost:~/Documents/SSHMonitor2.7$ git branch
* SSHMonitor3
anthony@ghost:~/Documents/SSHMonitor2.7$
```

**Comment by amboxer21** [comment]
SSHMonitor can now be interchangeably used between Python2 and Python3. This has been resolved in PR #25
